src/engine/Engine.ts

```diff
@@ -1045,19 +1049,20 @@ module ex {
       /**
        * Starts the internal game loop for Excalibur after loading
        * any provided assets. 
-       * @param loader  Optional resources to load before starting the main loop. Some [[ILoadable]] such as a [[Loader]] collection, 
-       * [[Sound]], or [[Texture]].
+       * @param loader  Optional [[ILoader]] to use to load resources. The default loader is [[Loader]], override to provide your own
+       * custom loader.
        */
-      public start(loader?: ILoadable) : Promise<any> {
+      public start(loader?: ILoader) : Promise<any> {
```

Comment:
This is technically a breaking API change. But it didn't make sense in order to separate drawing/update a Loader. Potentially, one way to allow the previous implementation is to accept an overload which still accepts `ILoadable` and then create the default `Loader`.